rust/src/container.rs

```diff
@@ -387,6 +398,20 @@ pub fn container_encapsulate(args: Vec<String>) -> CxxResult<()> {
         })
         .collect::<Result<_>>()?;
 
+    let package_structure = match &opt.ocimeta_json {
+        Some(ocimeta_json) => {
+            let path = std::path::Path::new(ocimeta_json);
```

Comment:
Because `Utf8Path` implements `AsRef<Path>`, we should be able to just directly pass the path as ` oci_spec::image::ImageManifest::from_file(&ocimeta_json)`.